docs/changelog/109071.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 109071
+summary: Better handling of multiple rescorers clauses with LTR
```

Comment:
Nitpick: Do we either have a GH issue to link to here or a slightly more descriptive summary for release notes?